### DISCUSSION TITLE
Documentação de boas práticas e recomendações gerais.

### DISCUSSION THREAD

**Discussion by JonasJs** [diego3g/rsxp-2023#18] [Ideas]
Seria interresante criamos um documento de boas praticas, regras, recomendações e passo a passo de como se iniciar as contribuições, acredito que esse documento ajudaria dar um norte para quem irá iniciar as contribuições no projeto.

Penso que seria principalmente recomendações de organização, padronizações de processos e talvez entre outros detalhes como formatos/padronizações de PRs para facilitar os reviews.

**Comment by d3vlopes** [comment]
Vai ser super importante tem um documento assim,  como se fosse um "contrato moral" entre cada contribuidor.

**Reply by viniseven** [reply]
Vai ter pessoas dentro dessa criação que vai ter boa experiência, mas outras com muito pouca, que vão mais acompanhar de perto pelo aprendizado, que pode ser ainda maior com, além das lives do diego, as dailys, e turmas que se juntarem para desenvolver, assim um poderá aplicar ensinamentos a outros e aprender juntos. Que pelo que creio é o que vai acontecer, vai ter pessoas dos variados níveis querendo participar. O que é bom.  Então por ter pessoas com muitas e outras com menos experiências justamente a isso, seria recomendável ter uma guia de boas práticas e padronizações, tanto pra quem já programa a um bom tempo quanto pra quem programa a pouco tempo. 
* Padronizações de código
* Nomenclaturas de funções, componentes
* Estrutura de pastas e name de arquivos, e por aí vai...

**Comment by FelipeQueiroz** [comment]
Cara eu gosto bastante da padronização de Pull Request da [Brasil API](https://github.com/BrasilAPI/BrasilAPI/blob/main/CONTRIBUTING.md) 

Penso que podemos fazer bem similar, só que teriamos que mudar as configurações para iniciar o projeto, já que vai ser mobile.

**Comment by Rodrigo001-dev** [comment]
Gelere, uma duvida, qual a recomendação de vocês para a documentação das taks?
Alguém teria alguma recomendação?
